metadata_service/config.py

```diff
@@ -27,6 +27,17 @@ class Config:
 
     IS_STATSD_ON = False
 
+    # Used to differentiate tables with other entities in Atlas. For more details:
+    # https://github.com/lyft/amundsenmetadatalibrary/blob/master/docs/proxy/atlas_proxy.md
+    ATLAS_TABLE_ENTITY = 'Table'
```

Comment:
Are these something that needs to be configurable? Or is this the one expected to change in upcoming version?

Reply:
These are the settings that should be configurable, as it can vary based on the atlas deployments and configurations within an organization.